terraform/infrastructure/azure/modules/scale_set/variables.tf

```diff
@@ -96,3 +96,14 @@ variable "secure_boot" {
   default     = false
   description = "Whether to deploy the cluster nodes with secure boot."
 }
+
+variable "marketplace_image" {
```

Comment:
we should expose this in our `azure-constellation` module too.

Reply:
I don't think so - the module will be deprecated with the next release and we will want new users to use the provider, to which I added support for the marketplace images.